elytron/src/test/java/org/wildfly/extension/elytron/TlsTestCase.java

```diff
@@ -644,6 +648,53 @@ public void testOcspSimple() {
     }
 
+    @Test
+    public void testOcspStaplingServerSimple() {
+        ModelNode operation = new ModelNode();
+        operation.get(ClientConstants.OP_ADDR).add("subsystem", "elytron").add(ElytronDescriptionConstants.SERVER_SSL_CONTEXT, INIT_TEST_SERVER_SSL_CONTEXT);
+        operation.get(ClientConstants.OP).set(ClientConstants.ADD);
+        operation.get(ElytronDescriptionConstants.KEY_MANAGER).set("ServerKeyManager");
+        operation.get(ElytronDescriptionConstants.PROVIDERS).set("ManagerProviderLoader");
+        Assert.assertEquals(SUCCESS, services.executeOperation(operation).get(OUTCOME).asString());
+
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING);
+        operation.get(ClientConstants.VALUE).set(Collections.emptySet());
+        Assert.assertEquals(SUCCESS, services.executeOperation(operation).get(OUTCOME).asString());
+
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING + "." + ElytronDescriptionConstants.RESPONSE_TIMEOUT);
+        operation.get(ClientConstants.VALUE).set(2500);
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING + "." + ElytronDescriptionConstants.CACHE_SIZE);
+        operation.get(ClientConstants.VALUE).set(512);
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING + "." + ElytronDescriptionConstants.CACHE_LIFETIME);
+        operation.get(ClientConstants.VALUE).set(7200);
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING + "." + ElytronDescriptionConstants.RESPONDER_URI);
+        operation.get(ClientConstants.VALUE).set("http://localhost:8080/ocsp");
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING + "." + ElytronDescriptionConstants.RESPONDER_OVERRIDE);
+        operation.get(ClientConstants.VALUE).set(true);
+        operation.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
+        operation.get(ClientConstants.NAME).set(ElytronDescriptionConstants.OCSP_STAPLING + "." + ElytronDescriptionConstants.IGNORE_EXTENSIONS);
+        operation.get(ClientConstants.VALUE).set(true);
+        Assert.assertEquals(SUCCESS, services.executeOperation(operation).get(OUTCOME).asString());
+    }
+
+//    @Test
```

Comment:
This is commented out because when executing these commands from a test, the following line fails when creating the client context. 
https://github.com/wildfly/wildfly-core/blob/e409c5b6392b21b4f6732f74df4f2c4b398343ab/controller/src/main/java/org/jboss/as/controller/OperationContextImpl.java#L629
However, when running this add command from the management CLI, this does not fail. I am not sure if that failure is expected due to the set up of the tests.